src/applications/proxy-rewrite/local-proxy-rewrite.js

```diff
@@ -26,7 +26,18 @@ function injectLocalBundle() {
       const vaPageResponse = await fetch(vaGovUrl);
       const vaPageHtml = await vaPageResponse.text();
 
-      vaGovCache[vaGovUrl] = vaPageHtml.replace(prodBucketRegex, '');
+      vaGovCache[vaGovUrl] = vaPageHtml
```

Comment:
webpack is acting as the proxy in this text- this is just overwriting the existing paths before they're served to the browser.

Reply:
sneaky!